randovania/games/prime1/layout/prime_configuration.py

```diff
@@ -66,15 +66,24 @@ def dangerous_settings(self) -> List[str]:
         result = super().dangerous_settings()
 
         if self.shuffle_item_pos:
-            result.append("Shuffled item position")
+            result.append("Shuffled Item Position")
 
         if not self.qol_game_breaking:
             result.append("Missing Game Breaking Fixes")
         
         if self.room_rando != RoomRandoMode.NONE:
-            result.append("Room randomizer")
-        
+            result.append("Room Randomizer")
+
         if self.large_samus:
             result.append("Large Samus")
 
+        if self.superheated_probability > 0:
+            result.append("Extra Superheated Rooms")
```

Comment:
Should add a test that calls `dangerous_settings` on a preset with all these options enabled.

Reply:
To test that a preset with all dangerous options is flagged as dangerous? I feel like that's not in jeopardy of failing ever

Reply:
The test I added was in response to this comment.